src/pages/auth/Auth.jsx

```diff
@@ -4,7 +4,7 @@ import "./auth.css";
 import { Login, SignUp } from "./components";
 
 export function Auth() {
-  useDocTitle("Auth - SmashCart - Manoj Sarna");
+  useDocTitle("Auth - SmashTube - Manoj Sarna");
   const [authVal, setAuthVal] = useState(true);
```

Comment:
you could use the descriptive names or use comments to make the code readable. AuthVal doesn't convey anything